sdcm/nemesis.py

```diff
@@ -5151,6 +5151,35 @@ def disrupt_disable_binary_gossip_execute_major_compaction(self):
             self.target_node.restart_scylla_server()
             raise
 
+    def disrupt_rolling_restart_topology_coordinator_node(self):
```

Comment:
the name rolling restart doesn't related to what going on with this nemesis

it's multiple restarts of the same node.

rolling means, all of the cluster is being restarted (one by one, hence where the rolling comes from)

Reply:
@aleksbykov please update the PR headline and the commit headline, to match the name change